app/controllers/api/generic_events_controller.rb

```diff
@@ -56,5 +60,15 @@ def event_type
     def created_by
       current_user&.owner&.name || 'unknown'
     end
+
+    def event_specific_relationships
```

Comment:
Not a blocker: this method might be better suited living outside the controller where we can unit test it